Documentation/Api/RestApi.apib

```diff
@@ -35,15 +34,16 @@ require authentication.
 
         {
             "id": 241,
-            "loginToken": "2cfe100561473c6cdd99c9e2f26fa974"
+            "key": "2cfe100561473c6cdd99c9e2f26fa974",
+            "expiry": "2017-07-20T18:22:48+00:00"
         }
 
 + Response 401 (application/json)
 
     + Body
 
         {
-            "code": 1,
+            "code": 1500567098798,
             "message": "Not authorized",
             "description": "The user name and password did not match any existing user."
```

Comment:
Well, the consistency is that it's JSON, and that the HTTP status code represents the outcome if the request (success, not authorized, etc.). Apart from that, the client needs to know what data (fields) will be returned.